build/Dockerfile.addon.rhtap

```diff
@@ -11,6 +11,16 @@ RUN GOOS=${OS} \
     make build --warn-undefined-variables
 
 FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
+
+LABEL \
```

Comment:
these labels are generated by the script [here](https://redhat-internal.slack.com/archives/C04L50S5XM4/p1736965246968289).